src/components/InvertedFlatList/index.js

```diff
@@ -16,57 +13,58 @@ const propTypes = {
     }).isRequired,
 };
 
-// This is copied from https://codesandbox.io/s/react-native-dsyse
+// This is adapted from https://codesandbox.io/s/react-native-dsyse
 // It's a HACK alert since FlatList has inverted scrolling on web
-const InvertedFlatList = (props) => {
-    const ref = useRef(null);
+class InvertedFlatList extends React.Component {
```

Comment:
React.Component instead of Component. I'm guessing this is specifically for web?

Reply:
`Component` is just `React.Component` imported as `import {Component} from 'react'` 🤔  or are you asking something else here?

Reply:
Oh, ignore me here. I didn't know this was pointing to the same reference.